src/Seq.App.Slack/SlackReactor.cs

```diff
@@ -59,10 +55,23 @@ public class SlackReactor : Reactor, ISubscribeTo<LogEventData>
             IsOptional = true)]
         public string IconUrl { get; set; }
 
+        [SeqAppSetting(
+            DisplayName = "Proxy Server",
+            HelpText = "Proxy server to be used when making HTTPS request to slack api, uses default credentials",
+            IsOptional = true)]
+        public string ProxyServer { get; set; }
+
+        [SeqAppSetting(
+            DisplayName = "Dictionary serialiser string limit",
+            IsOptional = true,
+            HelpText = "If a property is a Dictionary, it's serialised as JSON and will be truncated if the resulting string is longer than this number")]
+        public int? JsonTrunateAt { get; set; } = null;
+
         private EventTypeSuppressions _suppressions;
 
         private static readonly IImmutableList<string> SpecialProperties = ImmutableList.Create("Id", "Host");
 
+        private static SlackApi slackApi;
```

Comment:
Nit, whitespace, and `_propertyName` convention

Reply:
fixed